trackpy/plots.py

```diff
@@ -298,8 +299,8 @@ def plot_traj(traj, colorby='particle', mpp=None, label=False,
         x = traj.set_index([t_column, 'particle'])['x'].unstack()
         y = traj.set_index([t_column, 'particle'])['y'].unstack()
         color_numbers = traj[t_column].values/float(traj[t_column].max())
-        print_update("Drawing multicolor lines takes awhile. "
-                     "Come back in a minute.")
+        logger.info("Drawing multicolor lines takes awhile. "
```

Comment:
Probably has to do with the event loop.  There are some issues on OSX in general.  We have asyncio tests that work perfectly on linux, but go in all the wrong order on mac.